Dockerfile

```diff
@@ -1,11 +1,17 @@
-FROM golang:latest
+FROM golang:latest as builder
 
-WORKDIR /opt/fortigate_exporter
+WORKDIR /build
 
 COPY . .
 RUN go get -v -t -d ./...
+RUN CGO_ENABLED=0 go build -o main .
+
+FROM scratch
+WORKDIR /opt/fortigate_exporter
 
-RUN go build -o main .
+COPY --from=builder /build/main .
+COPY ca-certificates.crt .
```

Comment:
I have been thinking about ca-certificates.crt. That seems like something a bit iffy for this repo to carry - it would quickly be out of date, and I not really a file you'd want to download from a random github repo I suppose. Is it possible to include Either https://curl.haxx.se/ca/cacert.pem or copy ca-certificates from the "builder"? I'm a bit of a Docker rookie.